src/node_modules/mask-factory.js

```diff
@@ -2,6 +2,35 @@
 
 module.exports = function maskFactory(maskDefinition) {
 	return function MaskDirective() {
+		var caret = {
+			set : function caretSet(element, pos) {
+				if (element[0].setSelectionRange) {
+					element[0].focus();
+					element[0].setSelectionRange(pos,pos);
```

Comment:
Hi guys!

I had this problem too, but i modify a little this PR... placing the (setTimeout function) encapsulating ==> element[0].setSelectionRange(pos,pos);

like this:
setTimeout(function () {
	element[0].setSelectionRange(pos,pos);
}, 0);

this work for me... regards